packages/braid-design-system/src/lib/components/MenuRenderer/MenuRenderer.tsx

```diff
@@ -380,30 +382,32 @@ export function Menu({
           placement === 'top' && styles.placementBottom,
         ]}
       >
-        <Box paddingY={styles.menuYPadding} className={styles.menuHeightLimit}>
-          {Children.map(children, (item, i) => {
-            if (isDivider(item)) {
-              dividerCount++;
-              return item;
-            }
+        <ScrollContainer direction="vertical" fadeSize="small">
+          <Box className={styles.menuHeightLimit}>
```

Comment:
The max height of the menu has increased slightly, and it looks like it's because the padding is applied on an element above where the height limit is set. Just worth noting but not an issue

Reply:
Actually, because the padding is outside now I can remove the padding from the max height calc in the stylesheet. Good pick up.